src/node_buffer.cc

```diff
@@ -518,7 +516,7 @@ void Copy(const FunctionCallbackInfo<Value> &args) {
   if (source_end - source_start > target_length - target_start)
     source_end = source_start + target_length - target_start;
 
-  uint32_t to_copy = MIN(MIN(source_end - source_start,
+  uint32_t to_copy = std::min(std::min(source_end - source_start,
                              target_length - target_start),
                              ts_obj_length - source_start);
```

Comment:
Tiniest of nits: can you line up the arguments?

Reply:
like this ?
```cpp
  uint32_t to_copy = std::min(
      std::min(source_end - source_start, target_length - target_start),
      ts_obj_length - source_start);
```

Reply:
@gengjiawen Either that or the way it was done before, I would say.